linchpin/provision/roles/openstack/tasks/provision_res_defs.yml

```diff
@@ -16,7 +16,7 @@
 
 - name: "Create name using uhash value"
   set_fact:
-    os_resource_name: "{{ os_resource_name + '-' + uhash + '-' }}"
+    os_resource_name: "{{ os_resource_name + 'default_delimiter' + uhash + 'default_delimiter' }}"
```

Comment:
I don't think you need quotes for default_delimiter variable

Reply:
yeah, we don't, it's string by default but putting in ' ' is the same, points us that its a custom var. Let's take it out if it's not necessary.